packages/dataviews-client/src/types.ts

```diff
@@ -10,6 +10,7 @@ export interface EndpointParam {
   enum?: string[]
   default?: string | boolean | number
   required?: boolean
+  values?: any
```

Comment:
Let's have just enum or values as those do the same thing and seems to be duplicated, we need to agree with BE in which property we choose and remove one of them.